.github/ISSUE_TEMPLATE/1_bug_report.yaml

```
@@ -46,7 +21,7 @@ body:
       placeholder: "# Insert code here"
```

```suggestion
    placeholder: |
      ```python
      # Sample code to reproduce the problem
      ```
```